src/stories/components/Header/Header.tsx

```diff
@@ -64,20 +69,20 @@ const Header: React.FC = () => {
         </ContainerLogoSelect>
 
         <Navigation>
-          {menuItems.map((item: MenuType) => (
-            <Link href={item.link} passHref key={item?.title}>
+          {Object.keys(menuItems).map((item) => (
```

Comment:
I'll recommend to iterate the object with both it's key and value so we don't have to acces the value through the key evry time, also I wouldn't use item to reffer to the key of the item but for the value itself, since this way seems counterintuitive, normally it'll go: ```item = object[key]``` instead of ```something=object[item]```

```suggestion
          {Object.entries(menuItems).map(([key, item ]) => (
          // here we can use item instead of menuItems[key] which is cleaner and also provides the target value directly
```

In case you're never going to need the key name, you could even go further and do 
```suggestion
          {Object.values(menuItems).map((item) => (
          // here you can do item.something with item being the value of that entry on each iteration
```

Reply:
you are right, done